.travis.yml

```diff
@@ -5,6 +5,12 @@ language: dart
 services:
   - "redis-server"
 
+addons:
+  chrome: stable
+env:
+  global:
+    - CHROME_BIN=/usr/bin/google-chrome
```

Comment:
Let's detect this in our test framework...

Reply:
Keeping it simple for now: I've hard-coded the path in the code, and added a TODO to make this better.